include/flang/ISO_Fortran_binding.h

```diff
@@ -112,6 +107,19 @@ typedef struct CFI_dim_t {
   CFI_index_t sm; /* memory stride in bytes */
 } CFI_dim_t;
 
+#ifdef __cplusplus
+// C++ does not support flexible array.
+// The below structure emulate a flexible array. This structure does not take
```

Comment:
"The structure below emulates a flexible array."

Reply:
Thanks